.eslintrc.js

```diff
@@ -71,5 +71,17 @@ module.exports = {
         jest: true,
       },
     },
+    {
+      files: ['types/**/*.{ts,tsx}'],
+      parser: '@typescript-eslint/parser',
+      plugins: ['@typescript-eslint/eslint-plugin'],
+      rules: {
+        '@typescript-eslint/no-unused-vars': 0,
```

Comment:
nit: ESLint [maps](https://eslint.org/docs/latest/user-guide/configuring/rules) 0, 1, 2 to "off", "warn", "error". So using "off" instead of 0 might be more readable.